src/object-properties.js

```diff
@@ -1,8 +1,29 @@
 const properties = {
+  /**
+   * @type {boolean}
+   */
   showTitles: false,
+  /**
+   * @type {string}
+   */
   title: '',
+  /**
+   * @type {string}
+   */
   subtitle: '',
+  /**
+   * @type {string}
+   */
   footnote: '',
+  /**
+   * @typedef {object}
+   */
+  style: {
+    /**
+     * @type {string}
+     */
+    label: '',
+  },
 };
```

Comment:
I would suggest adding `actions: []` here to make sure it exists when the object is created